gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcApplicationRepository.java

```diff
@@ -264,6 +264,11 @@ public Set<Application> findByIds(Collection<String> ids, Sortable sortable) thr
         }
     }
 
+    @Override
+    public Set<Application> findAll() throws TechnicalException {
+        return this.findAll(ApplicationStatus.values());
```

Comment:
I could also implement a findAll without the status and therefore without using this one. But I don't know if it's really necessary 🤷‍♂️

Reply:
Looks ok to me like this